etc/hyperchains/docker-compose-hyperchain-template

```diff
@@ -3,24 +3,27 @@ networks:
   zkstack:
     driver: bridge
 volumes:
```

Comment:
is it possible to validate this docker-compose template in `CI` to ensure vars configured inside works ?

Reply:
This is the idea indeed. @hatemosphere will add some CI pipeline to spin it up with  "zk stack demo skip-env-setup"

Reply:
we have a plan to run basic tests indeed, however let's not do it in this PR